Styles/main.css

```diff
@@ -3,17 +3,19 @@
 @import url(/Styles/general.css);
 @import url(/Templates/Buttons/buttons.css);
 @import url(/Styles/utilities.css);
+
 :root {
     font-size: 16px;
 }
 
-body{
+body {
     font-family: 'Poppins', sans-serif;
 }
 
-.prim-color{
+.prim-color {
     color: var(--primary);
 }
```

Comment:
Any reason you created property of a single css variable? Could have been avoided as you are again specifying variable in line 76.

Reply:
Yeah, had to use it on the Header, was initially using inline styling because it was only a single property, changed it to this after thinking it over a bit. Should I switch it back to inline or just keep it as is?

> Any reason you created property of a single css variable? Could have been avoided as you are again specifying variable in line 76.

Reply:
That shouln't be a problem. I think you should be able to use css variables anywhere if they are getting linked at a single point and you are using that single point as your main stylesheet